### DISCUSSION TITLE
How to run GTA V on Steam Deck

### DISCUSSION THREAD

**Discussion by MarcinRNowak** [Heroic-Games-Launcher/HeroicGamesLauncher#1519] [Q&A]
Hello, 

I was trying to install GTA V and RDR2 from the Epic store on Steam Deck, but both have the same error. (TypeError: Cannot read properties of null ('bin')). There is only this one line in the log file.

I read somewhere that I need to install the Rockstar launcher. Is this correct? Is this going to solve the issues?

Thanks,
Marcin

**Comment by flavioislima** [comment]
This is a Heroic bug. Just change the wine version to any other and get back to the one you were using and it will update the setting.
